src/test/java/millom/sandbox/mapper/NasaMapperTest.java

```diff
@@ -0,0 +1,247 @@
+      + "        \"sunrise_sunset_desc_en\": \"\\n\\t\\t\\t\\tThe duration of a Martian day (sol) is about 24 hours and 40 minutes. The duration\\n\\t\\t\\t\\tof daylight varies along the Martian year, as on Earth.\\n\\t\\t\\t\",\n"
+      + "        \"sunrise_sunset_desc_es\": \"\\n\\t\\t\\t\\tUn d�a marciano (sol) dura aproximadamente 24 horas y 40 minutos. La duraci�n\\n\\t\\t\\t\\tdel d�a y de la noche var�a a lo largo del a�o, como ocurre en la Tierra.\\n\\t\\t\\t\"\n"
+      + "    },\n"
+      + "    \"soles\": [\n"
+      + "        {\n"
+      + "            \"id\": \"3345\",\n"
+      + "            \"terrestrial_date\": \"2022-07-03\",\n"
+      + "            \"sol\": \"3522\",\n"
+      + "            \"ls\": \"258\",\n"
+      + "            \"season\": \"Month 9\",\n"
+      + "            \"min_temp\": \"-67\",\n"
+      + "            \"max_temp\": \"-10\",\n"
+      + "            \"pressure\": \"870\",\n"
+      + "            \"pressure_string\": \"Higher\",\n"
+      + "            \"abs_humidity\": \"--\",\n"
+      + "            \"wind_speed\": \"--\",\n"
+      + "            \"wind_direction\": \"--\",\n"
+      + "            \"atmo_opacity\": \"Sunny\",\n"
+      + "            \"sunrise\": \"05:52\",\n"
+      + "            \"sunset\": \"18:09\",\n"
+      + "            \"local_uv_irradiance_index\": \"High\",\n"
+      + "            \"min_gts_temp\": \"-72\",\n"
+      + "            \"max_gts_temp\": \"8\"\n"
+      + "        },\n"
+      + "        {\n"
+      + "            \"id\": \"3344\",\n"
+      + "            \"terrestrial_date\": \"2022-07-02\",\n"
+      + "            \"sol\": \"3521\",\n"
+      + "            \"ls\": \"258\",\n"
+      + "            \"season\": \"Month 9\",\n"
+      + "            \"min_temp\": \"-69\",\n"
+      + "            \"max_temp\": \"-14\",\n"
+      + "            \"pressure\": \"882\",\n"
+      + "            \"pressure_string\": \"Higher\",\n"
+      + "            \"abs_humidity\": \"--\",\n"
+      + "            \"wind_speed\": \"--\",\n"
+      + "            \"wind_direction\": \"--\",\n"
+      + "            \"atmo_opacity\": \"Sunny\",\n"
+      + "            \"sunrise\": \"05:51\",\n"
+      + "            \"sunset\": \"18:08\",\n"
+      + "            \"local_uv_irradiance_index\": \"High\",\n"
+      + "            \"min_gts_temp\": \"-77\",\n"
+      + "            \"max_gts_temp\": \"8\"\n"
+      + "        }\n"
+      + "]\n"
+      + "}";
+  public static final String RESPONSE_PROPERTY_MISMATCH="{\n"
```

Comment:
Public?
It might also be a good idea to move this big Strings to a helper class maybe?

Reply:
moved 
public static final String RESPONSE_PROPERTY_MISMATCH
to helper class **JsonUtility.java**